src/cascadia/TerminalControl/InteractivityAutomationPeer.idl

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.Terminal.Control
+{
+    [default_interface] runtimeclass InteractivityAutomationPeer :
+        Windows.UI.Xaml.Automation.Peers.AutomationPeer,

```

Comment:
So, I was trying to avoid using as much XAML as possible in the Interactivity layer, since I wanted that to ultimately be UI framework independent. Most of the rest of the UIA stuff is however, _real_ XAML-y. So I can't be totally pure, no-Windows.UI.Xaml here unfortunately. I think it's still a good idea to try and move away from XAML for these bits as much as possible. 

On the latest commit, I've fixed the signaling for JAWS. That means we definitely don't need the whole `FrameworkElementAutomationPeer`, so I'd rather use as little XAML as we can